### ISSUE TITLE
Data ambiguities

### ISSUE THREAD

**Issue by PaulLiuYZ** [OpenMeshLab/MeshXL#11]
For the ground truth mesh models, are there any specific rules for triangulation? For example, a rectangular plane may be represented by different numbers and different ways of triangular faces, there may be ambiguity here?

**Comment by ch3cook-fdu** [comment]
The main concept behind our MeshXL is that **a 3D mesh with a given triangulation can be modeled into a one-and-only ordered coordinates sequence**. Therefore, we do not design extra rules for triangulation as long as they are valid 3D meshes. In our project, different triangulations also serve as a potential data augmentation strategy, as **our MeshXL treat different topologies as different training samples**. We adopt planar decimation (with different threshold) to simplify complex 3D meshes, and rotate 3D meshes (and create ordered coordinate sequences after) to create more training samples. 

In short, different topologies for a 3D mesh are not considered ambiguities in our project. It is somewhat similar to `ColorJitter`, `RandomHorizontalFlip`, and `Resize` operations for images.

**Comment by PaulLiuYZ** [comment]
Get it! Thanks for your reply!
